utils/download_filtered.js

```diff
@@ -26,95 +36,121 @@ function downloadFiltered(config, callback) {
       minTime: 6000
     };
     const limiter = new Bottleneck(options);
-    const callbackOnLastOne = () => {
+    const done = () => {
       if (limiter.empty()) {
         callback();
       }
     };
-    files.map(file => {
-      limiter.submit(downloadSource, targetDir, file, callbackOnLastOne);
+    sources.map(source => {
```

Comment:
Should filter sources with error first, otherwise this will download a file with `undefined` as url.

Reply:
resolved via rebase